### ISSUE TITLE
Steam Deck Flatpak Fails to Launch Any Game with WINE

### ISSUE THREAD

**Issue by BigOakley** [flathub/net.lutris.Lutris#221]
**Describe the problem**
After installing any game that uses the WINE runner, UI buttons that would launch the game fail to do so.

**Is it reproducible with native Lutris?**
I have not experienced this on my desktop installation, but the Steam Deck has a read only filesystem and I can't install the Lutris client natively

**Steps to reproduce**
1. Install an application with the WINE runner. I used the Epic Games Store using the Lutris script
2. Attempt to launch the game using the play button in the UI

**Expected behavior**
What you expected to happen.

I expect the game, Epic Game Store, to open and allow me to install or launch other games.

**Actual behavior**
What actually happens.

I get a traceback message that is always the same, regardless of the application or the UI interaction with the WINE prefix.

```
2022-03-22 17:58:20,101: Unable to load libGLX_nvidia.so.0
2022-03-22 17:58:20,102: Unable to locate libGLX_nvidia
Traceback (most recent call last):
  File "/app/lib/python3.9/site-packages/lutris/exceptions.py", line 57, in wrapper
    return function(*args, **kwargs)
  File "/app/lib/python3.9/site-packages/lutris/game.py", line 420, in configure_game
    gameplay_info = self.get_gameplay_info()
  File "/app/lib/python3.9/site-packages/lutris/game.py", line 398, in get_gameplay_info
    gameplay_info = self.runner.play()
  File "/app/lib/python3.9/site-packages/lutris/runners/wine.py", line 825, in play
    launch_info = {"env": self.get_env(os_env=False)}
  File "/app/lib/python3.9/site-packages/lutris/runners/wine.py", line 749, in get_env
    env = super(wine, self).get_env(False)
  File "/app/lib/python3.9/site-packages/lutris/runners/runner.py", line 186, in get_env
    runtime_env = self.get_runtime_env()
  File "/app/lib/python3.9/site-packages/lutris/runners/wine.py", line 789, in get_runtime_env
    for proton_path in get_proton_paths():
  File "/app/lib/python3.9/site-packages/lutris/util/wine/wine.py", line 49, in get_proton_paths
    for path in _iter_proton_locations():
  File "/app/lib/python3.9/site-packages/lutris/util/wine/wine.py", line 38, in _iter_proton_locations
    for path in [os.path.join(p, "common") for p in steam().get_steamapps_dirs()]:
  File "/app/lib/python3.9/site-packages/lutris/runners/steam.py", line 243, in get_steamapps_dirs
    library_config = self.get_library_config()
  File "/app/lib/python3.9/site-packages/lutris/runners/steam.py", line 149, in get_library_config
    return read_library_folders(self.steam_data_dir)
  File "/app/lib/python3.9/site-packages/lutris/util/steam/config.py", line 161, in read_library_folders
    library["libraryfolders"].pop("contentstatsid")
KeyError: 'contentstatsid'
```

**Flatpak info**
`flatpak --version` `flatpak 1.12.4`

`flatpak --gl-drivers` `default host`

**Comment by A6GibKm** [comment]
Thanks for your report, but I would suggest you try your luck at https://github.com/lutris/lutris/. The traceback you included seems to be rather useful for a quick debug on their side.

If all else fails I could try during the week to take a look at it.

**Comment by A6GibKm** [comment]
One question though, do you have the flatpak for org.gnome.Platform.Compat.i386? Note that today I updated the runtime to GNOME 42, so it should be org.gnome.Platform.Compat.i386//42 if you updated today, or //41 if you are still on yesterday version of the flatpak.

**Comment by BigOakley** [comment]
> One question though, do you have the flatpak for org.gnome.Platform.Compat.i386 installed? Note that today I updated the runtime to GNOME 42, so it should be org.gnome.Platform.Compat.i386//42 if you updated today, or //41 if you are still on yesterday version of the flatpak.
> 
> You can check with
> 
> ```
> flatpak list |grep org.gnome.Platform.Compat
> ```
```
i386    org.gnome.Platform.Compat.i386          41      flathub user
i386    org.gnome.Platform.Compat.i386          42      flathub user
```

I just updated to 42 after reading this response, and got the same result. I went and checked on the main Lutris project site and found this issue [4141](https://github.com/lutris/lutris/issues/4141).

And this issue points to this recent commit [072e72a] (https://github.com/lutris/lutris/commit/072e72a4aefd91101b79dd05d8ce9f100a4b6b0c).

This appears to be the same traceback, and looks like it might resolve the issue that I am running in to on the Steam Deck

**Comment by A6GibKm** [comment]
Thanks for looking into it, for the looks of it we could have a lutris release soon. Note that if this is a widespread issue we can patch it here for the time being.

**Comment by BigOakley** [comment]
I am in no rush here. I can wait and see if a new release fixes the issue.

**Comment by codebam** [comment]
I have this issue too. Fedora Silverblue 35. Installing a wine runner in the flatpak and launching a game gives me:

`[Errno 16] Device or resource busy: '/var/home/codebam/.wine'`

**Comment by A6GibKm** [comment]
Ill do a patch tomorrow then

**Comment by strycore** [comment]
Fixed in lutris 0.5.10

**Comment by A6GibKm** [comment]
Thanks.
